api-report/tree2.api.md

```diff
@@ -503,6 +520,9 @@ export class FieldKind<TEditor extends FieldEditor<any> = FieldEditor<any>, TMul
 // @alpha
 export type FieldKindIdentifier = Brand<string, "tree.FieldKindIdentifier">;
 
+// @public (undocumented)
+export const FieldKindIdentifier: TUnsafe<FieldKindIdentifier>;
```

Comment:
I had hoped all of these exports could be fixed by using `export type` appropriately, but it looks like [even that is an outstanding api-extractor issue](https://github.com/microsoft/rushstack/issues/3616).

IMO this is annoying enough that it's worth *not* adopting the "export type + const representing the schema of that type* organization strategy. But I'd like consensus on what convention we should use instead--I could imagine a few.